### ISSUE TITLE
Add Node 6 support

### ISSUE THREAD

**Issue by vladikoff** [dannycoates/scrypt-hash#12]
Tried installing this on Node 6.4.0 and getting:

```
➜  scrypt-hash git:(master) npm i 

> scrypt-hash@1.1.13 install /Users/vladikoff/dev/scrypt-hash
> node-gyp rebuild

gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
  CC(target) Release/obj.target/scrypt/crypto_scrypt-sse.o
  CXX(target) Release/obj.target/scrypt/node_scrypt.o
In file included from ../node_scrypt.cc:3:
../node_modules/nan/nan.h:590:20: error: no type named 'GCEpilogueCallback' in 'v8::Isolate'
      v8::Isolate::GCEpilogueCallback callback
      ~~~~~~~~~~~~~^
../node_modules/nan/nan.h:596:20: error: no type named 'GCEpilogueCallback' in 'v8::Isolate'
      v8::Isolate::GCEpilogueCallback callback) {
      ~~~~~~~~~~~~~^
../node_modules/nan/nan.h:601:20: error: no type named 'GCPrologueCallback' in 'v8::Isolate'
      v8::Isolate::GCPrologueCallback callback
      ~~~~~~~~~~~~~^
../node_modules/nan/nan.h:607:20: error: no type named 'GCPrologueCallback' in 'v8::Isolate'
      v8::Isolate::GCPrologueCallback callback) {
      ~~~~~~~~~~~~~^
4 errors generated.
make: *** [Release/obj.target/scrypt/node_scrypt.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/vladikoff/.nvm/versions/node/v6.4.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:204:12)
gyp ERR! System Darwin 15.6.0
gyp ERR! command "/Users/vladikoff/.nvm/versions/node/v6.4.0/bin/node" "/Users/vladikoff/.nvm/versions/node/v6.4.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/vladikoff/dev/scrypt-hash
gyp ERR! node -v v6.4.0
gyp ERR! node-gyp -v v3.3.1
gyp ERR! not ok 

npm WARN scrypt-hash@1.1.13 license should be a valid SPDX license expression
npm ERR! Darwin 15.6.0
npm ERR! argv "/Users/vladikoff/.nvm/versions/node/v6.4.0/bin/n
```

**Comment by dannycoates** [comment]
fixed in 9658f724a10a9a10ed4ba381648eabdbf61a05cd

**Comment by vladikoff** [comment]
Thanks so much!
